Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.h

```diff
@@ -28,5 +28,5 @@ static NSString * Githash = @"";
 @import WebKit;
 
 @interface ISVungleAdapter : ISBaseAdapter
-
+- (CGFloat)getAdaptiveHeightWithWidth:(CGFloat)width;
```

Comment:
what is this for? i dont see its being used any where?

Reply:
Thanks for catching. I forgot to remove.

fyi:
`getAdaptiveHeightWithWidth` is the API to support "adaptive" banner size.
With overriding this API, when a pub calls public API `CGFloat adaptiveHeight = [ISBannerSize getMaximalAdaptiveHeightWithWidth:width];`,  IS SDK calls this method and grab Vungle's size for adaptive case. So, the pub can set the "right" height and request banner.

But, now, we don't need to support adaptive, we decided to remove this.

Reply:
As I discussed with @ManojBudumuru-vungle, I updated code to send `adapterBannerDidFailToShowWithError` or `adapterBannerDidFailToLoadWithError` based on the ad load status.